package.json

```diff
@@ -43,6 +43,8 @@
   },
   "devDependencies": {
     "@dcl/eslint-config": "^1.1.12",
+    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
```

Comment:
All extra deps in this PR are to allow the Wearable Preview to be used by vite. The wearable preview depends on process.env to be available but vite handles it differently. This is done in the profile dapp as well https://github.com/decentraland/profile/blob/main/vite.config.ts